gm4_mountaineering/data/gm4_mountaineering/functions/ski_effects/downhill_ski_speed/check_downhill_ski_speed.mcfunction

```diff
@@ -0,0 +1,11 @@
+# Checks that the player still travels downhill (>=2 blocks)
+# @s = player with skiing tag and having fallen >=2 blocks
+# at @s
+# run from main
```

Comment:
You have a few of these headers with the wrong `run from`, and I found some as well that said they ran for "a player wearing skis" but actually ran for all players, make sure to carefully check these as a wrong header makes it easy to get confused on why things are not working as you think they should.